noir-projects/noir-protocol-circuits/crates/types/src/abis/accumulated_data/private_accumulated_data_builder.nr

```diff
@@ -100,8 +105,8 @@ impl PrivateAccumulatedDataBuilder {
         }
 
         revertible_builder.new_l2_to_l1_msgs =  self.new_l2_to_l1_msgs;
-        revertible_builder.encrypted_logs_hash = self.encrypted_logs_hash;
-        revertible_builder.unencrypted_logs_hash = self.unencrypted_logs_hash;
+        revertible_builder.encrypted_logs_hashes = self.encrypted_logs_hashes;
+        revertible_builder.unencrypted_logs_hashes = self.unencrypted_logs_hashes;
```

Comment:
True but I'm not sure how to handle the preimages length! Since we don't hash inside the circuit, the circuit doesn't 'know' the length of each log yet, so we can't split the length up. We do always recombine and sum the lengths anyway in `CombinedAccumulatedData`, so maybe that's fine for now?

Reply:
For now I split the logs, and added a TODO to do the same for the lengths once 1165 is complete. It doesn't seem to break anything, since lengths are re-summed in `combine` anyway.